src/countItems.js

```diff
@@ -4,20 +4,33 @@
  * values are the number of times that string appears.
  *
  * @example
+ *  * countItems(['apple', 'goat', 'apple', 'potato']);
  * // returns { 'apple': 2, 'goat': 1, 'potato': 1 }
- * countItems(['apple', 'goat', 'apple', 'potato']);
  *
  * @param {string[]} array - An array of strings
  * @returns {object} An object containing the count of each
  *  string in the input array
  */
 function countItems(array) {
   // This is your job. :)
+  let counts = {};
+  for (let i = 0; i < array.length; i++) {
+    if (counts.hasOwnProperty(array[i]) === false) {
```

Comment:
Since `counts.hasOwnProperty` itself returns `true` or `false`, this can be:

```javascript
if (!counts.hasOwnProperty(array[i])) {
 // ...
} else {
  // ...
}
```